src/source/src/connector_source.rs

```diff
@@ -192,7 +192,13 @@ impl StreamSourceReader for ConnectorSourceReader {
                 *split_offset_mapping
                     .entry(msg.split_id.clone())
                     .or_insert_with(|| "".to_string()) = msg.offset.to_string();
-                events.push(self.parser.parse(content.as_ref(), &self.columns)?);
+                match self.parser.parse(content.as_ref(), &self.columns) {
```

Comment:
<img width="1225" alt="Screen Shot 2022-08-01 at 14 24 22" src="https://user-images.githubusercontent.com/6970676/182085321-f235f5b3-e1c0-4c4c-86cd-c73d2bef66ad.png">
We have actually one error logging for parsing error, for JSON parsing. So we can unify them into one.

Reply:
My point is that we need to print the original value for future debugging.